CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Updated `storybook` from `^8.2.4` to `^8.6.0`
+- Updated `react` and `react-dom` from `^18.3.1` to `^19.0.0`
```

Comment:
```suggestion
```
I don't think no changelog is required as both are  dev dependencies

Reply:
The peer dependency and any compatibility changes we made to the main file is what is outward facing that should be in the changelog.